src/css/main.css

```diff
@@ -88,13 +88,16 @@ body,
 
 .word {
   margin-bottom: 20px;
-  display: grid;
+  display: flex;
   justify-content: center;
-  grid-template-columns: repeat(auto-fit, 50px);
+  align-items: center;
 }
 
 .word__char {
   height: 50px;
+  width: 100%;
```

Comment:
You can just use `width: 50px` (no need for the max-width property as far as I understand 🙂 )